websites/H/Harmony/presence.ts

```diff
@@ -0,0 +1,71 @@
+const presence = new Presence({
+    clientId: "882003722270572574"
+  }),
+  browseTimestamp = Math.floor(Date.now() / 1000);
+
+let npTitle = "WeAreHarmony.net",
+  npArtist = "We play the biggest hits!",
+  npOnAir = "AutoDJ",
+  npListeners = 0;
+
+function fetchStats(): void {
+  fetch("https://staff.weareharmony.net/api/nowplaying")
+    .then(result => result.json())
+    .then(result => {
+      npTitle = result.song.title;
+      npArtist = result.song.artist;
+      npOnAir = result.onAir.name;
+      npListeners = result.listeners;
+    });
+}
+
+fetchStats();
+setInterval(fetchStats, 10000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logobg",
+    smallImageKey: "nobg_white"
+  };
+
+  if(
+    document.location.hostname === "weareharmony.net" &&
+    document.location.hash.startsWith("#/")
+  ) {
```

Comment:
handled automatically, not needed
```suggestion
  if(document.location.hash.startsWith("#/")) {
```